stanford_events_importer.install

```diff
@@ -856,10 +849,6 @@ function stanford_events_importer_update_7307(&$sandbox) {
       node_save($node);
     }
   }
-  // Increment our sandbox pass for the next go-round.
-  $sandbox['current'] = $sandbox['current'] + 1;
-  // Determine our progress percentage to pass it to the next round.
-  $sandbox['progress'] = $sandbox['current'] / $sandbox['max'];
-
+  $sandbox['#finished'] = 1 - count($sandbox['nids']) / $sandbox['count'];
```

Comment:
This is returning `float(0.016877637130802)`, so it *should* be going through another pass of the `$sandbox`, but it's exiting out. Not clear why.

Reply:
(That `0.01687...` value is due to having ~1,110 nodes to loop through)